app/assets/javascripts/timelines_select_boxes.js

```diff
@@ -29,6 +29,10 @@
 //requires 'autocompleter'
 
 jQuery(document).ready(function($) {
+  var formatSelection = function(item) {
```

Comment:
You do want `var` when declaring a new variable, by doing it you are binding that variable `formatSelection` to the anonymous `function($) { // [...]` above. 

If you had not done that, the variable would have been bound to the `global` scope (in a browser, that would be the `window` object). 

You could have also gone with the shorthand for declaring a function:

```javascript
function formatSelection(item) { // [...]
```

Personally, I find the declaration using `var` better, as it emphasizes on functions being first-class and declares the  scope in an explicit manner.